### ISSUE TITLE
Let's Make Landing Page Responsive :- Frontend First Issue 1. Making Landing Page Responsive in all screens 

### ISSUE THREAD

**Issue by Kali-Decoder** [Kali-Decoder/Eco_GRiD_H23#29]

**Comment by Kali-Decoder** [comment]
Hey, Nice to be part of the team go with the client folder and structure get familiar with that, and start work on the landing page now spot any glitch if it is there, and start work on the responsiveness. Any Problem feel free to ask.

**Comment by Nancy-30** [comment]
![image](https://github.com/Kali-Decoder/Eco_GRiD_H23/assets/99819484/a7e57970-5f9d-4f91-b7fa-44a1fe578eba)

This error is appearing on multiple systems upon typing ```npm install```.
Kindly check if this is a general issue.

**Comment by Kali-Decoder** [comment]
yes , there is issue of versions in modules got with this command npm install --force .

**Comment by Nancy-30** [comment]
It is working now, 
Thanks for the quick fix.

**Comment by KshitizSachan** [comment]
Working on this issue.
Question: is there any specific design we need to follow for mobile view or general responsiveness works.

**Comment by Kali-Decoder** [comment]
Hey Kshitiz Sachan, there are no specific designs check the alignments in the landing page.

**Comment by KshitizSachan** [comment]
ok

**Comment by Jatingupta9120** [comment]
assign this issue to me

**Comment by Kali-Decoder** [comment]
@Jatingupta9120 you can start work on it I will review then merge it. If it suits the team.
